src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/Solution/SolutionResult.cs

```diff
@@ -33,6 +33,6 @@ public SolutionResult(ISolution solution)
 
         public Sections Sections { get; }
 
-        public IEnumerable<string> FrameworkShortNames { get; }
+        public IEnumerable<string> Frameworks { get; }
```

Comment:
How difficult would it be to introduce a `FrameworkModel`/`FrameworkResult` class now that you've reworked the implementation?

We would probably just have `Id` and `ShortName` properties initially.

Reply:
Added new Models/Interface `(SolutionFramework, SolutionFrameworkDTO, ISolutionFramework) ` by following existing implementation